decomp2dbg/clients/gdb/gdb_client.py

```diff
@@ -246,10 +246,10 @@ def _handle_connect(self, args):
             err("You must provide a decompiler name when using this command!")
             return
 
-        if (args.base_addr_start and not args.base_addr_end) or (args.base_addr_end and not args.base_addr_start):
+        if (args.base_addr_start != None and args.base_addr_end == None) or (args.base_addr_end != None  and args.base_addr_start == None):
```

Comment:
Sorry to ask for a change one more time... but for any time you do a `None` comparison, it should use the `is` operator. 
```python
args.base_addr_start is not None
```

Would you be able to updates these again?